packages/wdio-sumologic-reporter/src/index.ts

```diff
@@ -127,7 +137,7 @@ export default class SumoLogicReporter extends WDIOReporter {
             return
         }
 
-        const logLines = this.unsynced.slice(0, MAX_LINES).join('\n')
+        const logLines: any = this.unsynced.slice(0, MAX_LINES).join('\n')
```

Comment:
`logLines` should be a string.
```suggestion
        const logLines = this.unsynced.slice(0, MAX_LINES).join('\n')
```

Reply:
If we keep logLines as string, the we get comiple issue at the below line.
https://github.com/webdriverio/webdriverio/pull/5998/files/6b229224c5c73449a71e98ec0cea2794d8075625#diff-e602a5e637e9e2d7a22c875a92a5b3f71bfe2200f96f054aa5a7454be8776bc5L141

Error message:
`Type 'string' is not assignable to type '{ [key: string]: any; } | undefined'.ts(2769)`

Reply:
Why does that complain here? We are newly defining this variable and it should receive the type from that it gets assigned to.

Reply:
ah it is complaining here:
```js
            const resp = await got(this.options.sourceAddress, {
                method: 'POST',
                json: logLines
            })
```

we should check the Sumo API then if it accepts a string. Haven't used the reporter is a while and it does seem odd that we pass in a string.

Reply:
we should check the Sumo API then  > where can i check this?

Reply:
For now i have updated it to accept json